tests/src/LinkParser.cpp

```diff
@@ -25,8 +25,27 @@ struct Case {
     }
 };
 
+
+struct SanitizeCheck
+{
+    QString testValue{};
+    QString expectedValue{};
+};
+
 TEST(LinkParser, parseDomainLinks)
 {
+    const QList<SanitizeCheck> sanitizeCases = {
+        { "(twitch.tv)", "twitch.tv"}
+    };
+
+    for (auto &c : sanitizeCases)
+    {
+        LinkParser p(c.testValue);
+         ASSERT_TRUE(p.result().has_value()) << c.testValue.toStdString();
+        const auto &r = *p.result();
+        ASSERT_EQ(c.expectedValue, r.host);
```

Comment:
Since `QString`s are 16bit, having a human-readable string is nice:

```suggestion
        ASSERT_EQ(c.expectedValue, r.host) << r.host.toString().toStdString();
```

Additionally, the `rest` should be checked as well.

Reply:
Added a "/foo" to the end of the url and checked rest